app/api/event_copy.py

```diff
@@ -126,6 +126,7 @@ def create_event_copy(identifier):
         db.session.commit()
 
     for user_role in user_event_roles:
+        user_role_id = user_role.id
```

Comment:
How will this fix the issue?

Reply:
I have no idea how it will fix the issue, even doing the print statement instead of that while testing, the issue was fixed.
So I saw the pattern in the above function, and repeated the same thing, and it worked.
They all have variable_name_id named unused variable.